### DISCUSSION TITLE
make tubeup auto delete uploaded files

### DISCUSSION THREAD

**Discussion by 111100001** [bibanon/tubeup#285] [Ideas]
had to get a 5 TB vm to upload files. if there is a workaround to auto delete downloads after uploading them that would be appreciated. i have minimal coding experience.

**Comment by brandongalbraith** [comment]
Artifact files are deleted after successfully uploading to the Internet Archive.

**Comment by vxbinaca** [comment]
Follow the submission guidelines we have no clue what you were doing to cause the problem. Closed.
